lbrynet/core/utils.py

```diff
@@ -28,10 +51,7 @@ def is_valid_blobhash(blobhash):
     """
     if len(blobhash) != blobhash_length:
         return False
-    for l in blobhash:
-        if l not in "0123456789abcdef":
-            return False
-    return True
+    return all(is_valid_hashcharacter(l) for l in blobhash)
```

Comment:
Though if you're going that far, why not go all the way:

    return len(blobhash) == blobhash_length and all(is_valid_hashcharacter(l) for l in blobhash)